pkg/infra/remotecache/database_storage.go

```diff
@@ -54,36 +54,40 @@ func (dc *databaseCache) internalRunGC() {
 
 func (dc *databaseCache) Get(ctx context.Context, key string) (interface{}, error) {
 	cacheHit := CacheData{}
-	session := dc.SQLStore.NewSession(ctx)
-	defer session.Close()
 
-	exist, err := session.Where("cache_key= ?", key).Get(&cacheHit)
+	var r interface{}
```

Comment:
the tiniest of nitpics: I believe if you initialize  a `cachedItem` here , you can use that on line 82 and skip initializing a second variable and continue to return `item.Val` at the end.